File: Js/duplicateChar.js
a = "abcdeafabd"

b = a.split("")

done = []
b.forEach(element => {
    if (!done.includes(element)) {
        occ = b.filter((e) => e == element).length
        console.log(element,"  ",occ)
        done.push(element)
    }
});

File: Python/duplicateChar.py
a = "abcdeafabd"

def duplicateChar(a):
    b = list(a)
    
    countDist = {}

    for i in b:
        if i in countDist:
            countDist[i] = countDist[i] + 1
        else:
            countDist[i] = 1

    print(countDist)

duplicateChar(a)